README.md

```diff
@@ -39,11 +49,11 @@ Create a docker container using Dockerfile in this repo and run the container in
 
 ### Build the docker image
 
-    docker build -t wigwag-build-env_${USER} --build-arg user=${USER} --build-arg group=${USER} --build-arg uid=$(id -u) --build-arg gid=$(id -g) .
+    docker build -t pelion-build-env_${USER} --build-arg user=${USER} --build-arg group=${USER} --build-arg uid=$(id -u) --build-arg gid=$(id -g) .
```

Comment:
Since docker commands can get quite long, I'd suggest posting them in a code block.

That way, users don't need to horzontally scroll documentation to read the entire command.

I would do something like:
```suggestion sh
    docker build \
      -t pelion-build-env_${USER} \
      --build-arg user=${USER} \
      --build-arg group=${USER} \
      --build-arg uid=$(id -u) \
      --build-arg gid=$(id -g) \
      .
```